components/AdminProfileForm/AdminProfileForm.module.css

```diff
@@ -0,0 +1,104 @@
+
+.page {
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    margin: 7%;
+    margin-top: 5%;
+    width:100%;
+    height: 100%;
+    background-color: #FAFAFA 100vh;
+}
+
+.pagetitle {
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    gap: 1%;
+}
+
+.namebar {
+    width: 100%;
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    margin-top: 1%;
+    margin-bottom: 2%;
+}
+
+.boxes {
+    width: 100%;
+    display: flex;
+    flex-direction: row;
+    justify-content: center;
+    margin-top: 2%;
+    gap: 5%;
+}
+
+.box {
+    padding: 4%;
+    justify-content: space-between;
+    align-items: center;
+    border: 2px solid var(--sapphire-light);
+    border-radius: 25px;
+    width: 100%;
+    background-color: white;
+}
```

Comment:
for the fields in this boxes, in editing state can u make sure the boxes don't affect the spacing of the other boxes. like right now all the boxes shift down when u press edit. would it be possible to keep everything static upon pressing edit, only having the boxes appear around editable fields?